doc/source/tokudb/tokudb_variables.rst

```diff
@@ -949,6 +959,29 @@ case there are bugs. Not for use in production.
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: Off
+
+Enables the fast updates feature, which substantially facilitates execution of
```

Comment:
Again, this is not an adequate description, as it will make users have this is an option enabled by default.